### ISSUE TITLE
При загрузке настроек сбиваются права доступа

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1495]
Originally reported on Google Code with ID 1495

```
Проверить чтобы при загрузке настроек, при обновлении существующих записей
не изменялись дескрипторы безопасности.

Решить: это нужно только для настроек? Или и для настроек и для загрузки
данных?
```

Reported by `gs1994` on 2009-08-05 12:24:17

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2009-08-05 12:37:29

**Comment by gsbelarus** [comment]
```
Раньше права доступа изменялись на -1. Убрали вообще редактирование прав доступа при

загрузке из потока (пусть БО сам разбирается с ними). 
```

Reported by `flakekun` on 2009-08-25 15:34:17
- **Status changed**: `Started`

**Comment by gsbelarus** [comment]
```
http://code.google.com/p/gedemin/issues/detail?id=1527
```

Reported by `flakekun` on 2009-08-27 11:31:52
- **Status changed**: `Duplicate`
- **Merged into**: #1527
